repairnator/kubernetes-support/README.md

```diff
@@ -0,0 +1,131 @@
+# Deploy a high avaliable Repairnator in Kubernetes
+
+![Design](K8sRepairnatorDesign.jpg)
+
+
+## Setup mongodb and ActiveMQ
```

Comment:
separate MongoDB and ActiveMQ in two different sections.

Move MongoDB at the end, it's optional, one can use a MongoDB that is not in K8

Reply:
This issue is addressed in the recent commits.